### DISCUSSION TITLE
Please Help me

### DISCUSSION THREAD

**Discussion by DuskGithub** [aclap-dev/video-downloadhelper#518] [Help and Q&A]
### Your question

我无法从任何网站上下载视频，它能够很好的识别出视频，并且我尝试了从7.0.0到9.0.0的所有可下载版本。

### Addon & CoApp version and details

 {
  "version": "9.0.1.13a1",
  "target": "mozilla",
  "channel": "beta",
  "page_url": "https://www.youtube.com/watch?v=NZAHrU40QUI&t=285s",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tcp @ 000001e4fccaf640] Connection to tcp://manifest.googlevideo.com:443 failed: Error number -138 occurred\r\n[in#1 @ 000001e4fb38ea80] Error opening input: Error number -138 occurred\r\nError opening input file https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1716224324/ei/5CxLZv-gGoq17OsP5c-zaA/ip/144.48.80.176/id/359007ad4e344142/itag/614/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/wft/1/sgovp/clen%3D438414000%3Bdur%3D2050.266%3Bgir%3Dyes%3Bitag%3D248%3Blmt%3D1689423526877166/rqh/1/hls_chunk_host/rr4---sn-oguelnsl.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/0d/mm/31,29/mn/sn-oguelnsl,sn-oguesn6d/ms/au,rdu/mv/m/mvi/4/pl/24/force_finished/1/initcwndbps/13523750/vprv/1/playlist_type/DVR/dover/13/txp/5319224/mt/1716202376/fvip/3/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,wft,sgovp,rqh,xpc,force_finished,vprv,playlist_type/sig/AJfQdSswRQIhAOAYGf5Cke6rTs-kHNHofwnzghj97n1YhE_3aDl40oMrAiBrY6Z2EkHwg_lZ4UDlEcPLqSemVzgGoLGHTY8xfUcaXQ%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AHWaYeowRQIhAIuzJJt7n4wWVTAbK7ZQq4Q7psA_Mev-B_bkEopW4YwPAiB7q0ltC1yZfQhCHsVv-yk9MgQFjF3ryCZljLB-VDZdIg%3D%3D/playlist/index.m3u8.\r\nError opening input files: Error number -138 occurred\r\n",
    "id": "coapp_failure_8914a7bd-d26b-4a3d-9f4a-bd93c47f1cb3",
    "downloadable_id": "downloadable_4523517240153639"
  }
}

**Comment by paulrouget** [comment]
Do you use a proxy?

**Reply by GrampaWildWilly** [reply]
Right.  I had no problem downloading this one with Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.13a1 beta, CoApp 2.0.19.  But I don't use a proxy.

**Reply by DuskGithub** [reply]
Yes, I use a proxy to access the network and currently use Clash. Will this prevent me from downloading videos?
